### DISCUSSION TITLE
Do we use the `wasm_to_bytes.js` script in `cli`?

### DISCUSSION THREAD

**Discussion by petarvujovic98** [near/near-sdk-js#259] [Ideas]
Like the title says do we use this script anywhere?
If not should we just delete it?

Also, if there are other scripts we do not use, should we remove those as well?

**Comment by volovyks** [comment]
Doc says it is used in `promiseBatchActionDeployContract()`, I believe we still need it. cc @ailisp 
@petarvujovic98 What else do you want to delete? If it is not used - we should do it.

**Reply by petarvujovic98** [reply]
@volovyks I think we are using the `includeBytes` macro for babel that we have in the build tools for that if I am not wrong...

The other one is `save-byte-code.js` which I also don't see called anywhere, but again I might be wrong.

**Reply by ailisp** [reply]
Yes it was a workaround to help user include contract in code, and superseded by includeBytes. save-byte-code was for jsvm, it's not relevant either.
